docs/user-guide/ui.md

```diff
@@ -2,48 +2,82 @@
 ---
 
 
-Kubernetes has a web-based user interface that allows you to deploy containerized
-applications to a Kubernetes cluster, troubleshoot them, and manage the cluster itself.
+The web-based user interface of Kubernetes (Dashboard) allows you to deploy containerized applications to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. You can use it for getting an overview of applications running on the cluster, as well as for creating or modifying individual Kubernetes resources and workloads, such as Daemon sets, Pet sets, Replica sets, Jobs, Replication controllers and corresponding Services, or Pods.
 
-By default, the Kubernetes Dashboard is deployed as a cluster addon. It is enabled by default in Kubernetes 1.2 clusters. Click [here](/docs/user-guide/ui-access/) to learn more about the Dashboard access.
+The Dashboard also provides information on the state of Pods, Replication controllers, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources, as well as your deployed containerized applications. You can also change the number of replicated Pods, delete Pods, and deploy new applications using a deploy wizard.
+
+By default, the Kubernetes Dashboard is installed as a cluster addon. It is enabled by default as of Kubernetes 1.2 clusters.
+
+* TOC
+{:toc}
 
-## Using the Dashboard
+## Accessing the Dashboard
 
-The Dashboard can be used to get an overview of applications running on the cluster, and to provide information on any errors that have occurred. You can also inspect your replication controllers and corresponding services, change the number of replicated Pods, and deploy new applications using a deploy wizard.
+Navigate in your Browser to the following URL:
+```
+https://<kubernetes-master>/ui
+```
+This redirects to the following URL:
+```
+https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
+```
+The Dashboard UI lives in the `kube-system` [namespace](/docs/admin/namespaces/), but shows all resources from all namespaces in your environment.
+
+If you find that you are not able to access the Dashboard, you can install and open the latest stable release by running the following command:

```

Comment:
Again, Dashboard used as a lone proper name, not as a qualifyer/adjective (like in the paragraph above - the Dashboard UI) or in a qualified way (eg. 'the Kubernetes Dashboard'). No 'the' in my opinion - despite what was in the original :)